docs/modules/Core.ts.md

```diff
@@ -40,7 +40,7 @@ Added in v1.0.0
 **Signature**
 
 ```ts
-export declare const main: Effect.Effect<unknown, unknown, void>
+export declare const main: unknown
```

Comment:
I'm not exactly sure why it can't infer, but I fixed this by annotating `main`.

Also, I fixed another bug with Config loading that I found as a result of this.